src/NServiceBus.AcceptanceTests/BestPractices/When_subscribing_to_command_bestpractices_disabled_on_endpoint.cs

```diff
@@ -7,6 +7,7 @@
 
     public class When_subscribing_to_command_bestpractices_disabled_on_endpoint : NServiceBusAcceptanceTest
     {
+        [Ignore("not supported via code first API")]
```

Comment:
Yea I hit that as well a while back. Seems like routing has its own set of convention checks